### PR TITLE
MSISDN++

### PR DESCRIPTION
**Pull request by cloveras**
- phone number format: MSIDN
- Some formatting and minor improvements

### CODE REVIEW

File: docs/epayment.v1.yml [hunk 1]
```diff
@@ -318,14 +321,15 @@ components:
           example: NATIVE_REDIRECT
           description: |-
             The flow for bringing to user to the Vipps Wallet payment confirmation screen.
-            If `userFlow` is `PUSH_MESSAGE`, a valid value for `$.customer.phoneNumber` is required.
+            If `userFlow` is `PUSH_MESSAGE`, a valid value for `customer.phoneNumber` is required.
         expiresAt:
           type: string
           example: '2023-02-26T17:32:28Z'
+          format: date-time
```
Comment (OFollan) [inline docs/epayment.v1.yml:328]: Sadly, we can't add this line of code without introducing a hard time in our Code generator 😢 
TLDR; 
This will mark the expiresAt property as a DateTime. The dotnet framework will then deserialize to a DateTime **before** the Regex is applied to the incoming string. Ergo, we have no way of verifying the incoming date in a nice way. 

We _could_ trust the framework in the deserialization, but e.g. because of [this](https://blog.nodatime.org/2011/08/what-wrong-with-datetime-anyway.html) I don't trust DateTime at all. Ending up with DateTimes where the `DateTimeKind` is `Unspecified` is a major source of bugs (which we also had in ePayment at some point 😄 )  

So I actually dropped this line of code on purpose.
The **real** fix would be to find another generator tool, but that is a separate story..

**Review by OFollan — CHANGES_REQUESTED** [review]

File: docs/epayment.v1.yml (hunk 1 above)
Comment (cloveras) [reply docs/epayment.v1.yml:328]: Ah! Will fix!

Comment (cloveras) [inline docs/epayment.v1.yml:328]:
```suggestion
```

**Review by OFollan — APPROVED** [review]
